packaging_automation/upload_to_package_cloud.py

```diff
@@ -116,13 +116,14 @@ def package_exists(package_cloud_token: str, repo_owner: str, repo_name: str, pa
     parser.add_argument('--package_cloud_api_token', required=True)
     parser.add_argument('--repository_name', required=True, choices=supported_repos)
     parser.add_argument('--output_file_path', required=True)
+    parser.add_argument('--exec_folder_path', required=True)
 
     args = parser.parse_args()
 
     multiple_return_value = upload_files_in_directory_to_package_cloud(args.output_file_path,
                                                                        args.platform,
                                                                        args.package_cloud_api_token,
-                                                                       args.repository_name)
+                                                                       args.repository_name, args.exec_folder_path)
```

Comment:
what is the usage of this parameter ?

Reply:
It is used for finding the current branch. If current branch is not main branch, packages are not pushed

Reply:
To verify if I understood correctly, is `exec_forder_path` to specify the branch we are working on ? 
Or is it another parameter that you explained ?

Reply:
exactly. We need to know the "packaging" directory we are working on to get the current branch. We will execute this code from pipeline. Otherwise I need to get the current branch from github which is a little bit tricky. But if you want I can get the branch name from github actions. 
Getting the branch name from github actions is  as below
https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions/58034787

Reply:
If it wouldn't be a big /complex change, I think "github actions approach" looks more clear

Reply:
Fixed